### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by scarblase**
- [ ] Header height is set in 1 place (for the links)
- [ ] Content is vertically centered (for any header height)
- [ ] CSS is used to show all letters in Uppercase (don't type them in HTML)
- [ ] Logo is an image wrapped with a link
- [ ] **CSS Variable** is used for a blue color
- [ ] Pseudo-element is used for a blue line below the active link
- [ ] Code follows all the [Code Style Rules ❗️](./checklist.md)

- [DEMO LINK](https://scarblase.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://scarblase.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -1,3 +1,78 @@
...
+  display: flex;
+}
+
+.nav_link:hover {
+  color: (var(--color));
+}
+
+.is-active {
+  position: relative;
+  color: #00acdc;
```
Comment (etojeDenys) [inline src/style.css:65]:
```suggestion
  color: var(--color);
```

**Review by etojeDenys — CHANGES_REQUESTED** [review]
image
you need to pass all the tests, feel free to ask for some help with them in the chat on the platform

File: readme.md [hunk 2]
```diff
@@ -27,8 +27,8 @@ The page should match the design Pixel Perfect: all the sizes, colors and distan
 
 ❗️ Replace `<your_account>` with your Github username and copy the links to `Pull Request` description:
 
-- [DEMO LINK](https://<your_account>.github.io/layout_moyo-header/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_moyo-header/report/html_report/)
+- [DEMO LINK](https://scarblase.github.io/layout_moyo-header/)
+- [TEST REPORT LINK](https://scarblase.github.io/layout_moyo-header/report/html_report/)
```
Comment (volodymyr-soltys97) [inline readme.md:31]: Add these links to the PR description 
image

File: src/index.html [hunk 3]
```diff
@@ -10,13 +10,107 @@
...
+        href="#home"
+        class="logo_link"
+      >
+        <img
+          src="images/logo.png"
+          class="logo_png"
+          alt="moyo_logo"
+        />
+      </a>
+      <nav class="nav">
```
Comment (volodymyr-soltys97) [inline src/index.html:43]:
```suggestion
      </a>
      
      <nav class="nav">
```

File: src/index.html [hunk 4]
```diff
@@ -10,13 +10,107 @@
...
+        <ul class="nav_list">
+          <li class="nav_item">
+            <a
+              href="#apple"
+              class="is-active nav_link"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav_item">
```
Comment (volodymyr-soltys97) [inline src/index.html:53]: Add the blank line between the links
```suggestion
          </li>
          
          <li class="nav_item">
```

File: src/style.css [hunk 5]
```diff
@@ -1,3 +1,78 @@
+html {
+  box-sizing: border-box;
+}
+
 body {
+  --color: #00acdc;
```
Comment (volodymyr-soltys97) [inline src/style.css:6]: Create a variable with root selector

File: src/style.css [hunk 6]
```diff
@@ -1,3 +1,78 @@
...
+
+  font-size: 12px;
+  box-sizing: border-box;
```
Comment (volodymyr-soltys97) [inline src/style.css:9]: Move box sizing to the * selector

File: src/style.css [hunk 7]
```diff
@@ -1,3 +1,78 @@
...
+  font-family: Roboto, sans-serif;
```
Comment (volodymyr-soltys97) [inline src/style.css:10]: Move font styles to the html selector

File: src/style.css [hunk 8]
```diff
@@ -1,3 +1,78 @@
...
+  background: white;
```
Comment (volodymyr-soltys97) [inline src/style.css:11]: This color is default
```suggestion
```

File: src/style.css [hunk 9]
```diff
@@ -1,3 +1,78 @@
...
+  margin: 0;
+}
+
+header {
```
Comment (volodymyr-soltys97) [inline src/style.css:15]: Use only class selector
```suggestion
.header {
```

File: src/style.css [hunk 10]
```diff
@@ -1,3 +1,78 @@
...
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding-left: 50px;
```
Comment (volodymyr-soltys97) [inline src/style.css:19]:
```suggestion
  padding: 0 50px;
```

File: src/style.css (hunk 1 above)
Comment (volodymyr-soltys97) [reply src/style.css:65]: Not fixed 
image

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Good job 👍 
Let's improve your code
You need to remove the margin right for the last link to pass all tests 
image

File: src/index.html (hunk 3 above)
Comment (OleziO) [reply src/index.html:43]: Not resolved

File: src/style.css (hunk 5 above)
Comment (OleziO) [reply src/style.css:6]: Move all variables to the :root selector

File: src/style.css (hunk 6 above)
Comment (OleziO) [reply src/style.css:9]: Not resolved

**Review by OleziO — CHANGES_REQUESTED** [review]
image
Fix links gap
image
Fix all test

===============

If you have any questions - feel free to ask them in fe_chat

File: src/index.html (hunk 3 above)
Comment (vadiimvooo) [reply src/index.html:43]: Still not fixed

File: src/style.css (hunk 5 above)
Comment (vadiimvooo) [reply src/style.css:6]: Still not fixed

File: src/style.css (hunk 6 above)
Comment (vadiimvooo) [reply src/style.css:9]: Still not fixed

File: src/style.css (hunk 8 above)
Comment (vadiimvooo) [reply src/style.css:11]: Still not fixed

File: src/style.css (hunk 9 above)
Comment (vadiimvooo) [reply src/style.css:15]: Still not fixed

File: src/style.css (hunk 10 above)
Comment (vadiimvooo) [reply src/style.css:19]: Still not fixed

**Review by vadiimvooo — CHANGES_REQUESTED** [review]
Good job, but you have to fix all previous comments. Please feel free to ask for help in fe_chat

**Review by anastasiiavorobiova — CHANGES_REQUESTED** [review]
Consider fixing previous comments

**Pull request closed without merging**